pkg/apis/aadpodidentity/v1/types.go

```diff
@@ -70,6 +70,18 @@ type AzureAssignedIdentity struct {
 	Status AzureAssignedIdentityStatus `json:"Status"`
 }
 
+//AzurePodIdentityException contains the pod selectors for all pods that don't require
+// NMI to process and request token on their behalf.
+
+//+k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
```

Comment:
Can you please add a README under the readme directory on how ExceptionList can be used.